python-sdk/nuscenes/map_expansion/arcline_path_utils.py

```diff
@@ -186,19 +190,97 @@ def discretize(arcline_path_3: Dict[str, Any],
     return discretization
 
 
-def discretize_lane(arcline_list: List[Dict[str, Any]],
+def discretize_lane(lane: List[ArcLinePath],
                     resolution_meters: float) -> List[Tuple[float, float, float]]:
```

Comment:
Can we use Pose type for the return hint?

Reply:
Yep, I missed that.